tests/test_models.py

```diff
@@ -91,35 +92,30 @@ def test_validate_social_profile_websites_empty(self):
         self.user.save()
         assert_equal(self.user.social['profileWebsites'], [])
 
-    def test_validate_social_valid_website_simple(self):
-        self.user.social = {'profileWebsites': ['http://cos.io/']}
-        self.user.save()
-        assert_equal(self.user.social['profileWebsites'], ['http://cos.io/'])
-
-    def test_validate_social_valid_website_protocol(self):
-        self.user.social = {'profileWebsites': ['https://definitelyawebsite.com']}
-        self.user.save()
-        assert_equal(self.user.social['profileWebsites'], ['https://definitelyawebsite.com'])
-
-    def test_validate_social_valid_website_ipv4(self):
-        self.user.social = {'profileWebsites': ['http://127.0.0.1']}
-        self.user.save()
-        assert_equal(self.user.social['profileWebsites'], ['http://127.0.0.1'])
+    def test_validate_social_profile_website_many_different(self):
+        with open('../website/static/urlValidatorTest.json') as urlTestData:
+            data = json.load(urlTestData)
```

Comment:
This doesn't look like it's actually doing anything other than loading test data. There are no assertions or anything.